File: app/src/main/java/com/febers/uestc_bbs/post/PostPresenter.java
/*
 * Created by Febers at 18-6-9 上午9:22.
 * Copyright (c). All rights reserved.
 * Last modified 18-6-9 上午9:21.
 */

package com.febers.uestc_bbs.post;

import android.util.Log;

public class PostPresenter extends PostContract.Presenter{

    private PostContract.View mView;
    private static final String TAG = "PostPresenter";

    public PostPresenter(PostContract.View view) {
        super(view);
        mView = view;
    }

    @Override
    public void getPosts(int position) {
        Log.i(TAG, "getPosts: " + position);
    }
}

File: app/src/main/java/com/febers/uestc_bbs/module/post/presenter/PostPresenter.kt
/*
 * Created by Febers at 18-6-13 下午5:48.
 * Copyright (c). All rights reserved.
 * Last modified 18-6-13 下午5:48.
 */

package com.febers.uestc_bbs.module.post.presenter

import android.util.Log

class PostPresenter : PostContract.Presenter {

    constructor(mView: PostContract.View): super(mView)

    override fun getPosts(position: Int) {
        Log.i(TAG, "getPosts: $position")
    }

    companion object {
        private val TAG = "PostPresenter"
    }
}
